components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/src/main/java/org/wso2/carbon/identity/claim/metadata/mgt/ClaimMetadataManagementServiceImpl.java

```diff
@@ -337,6 +339,13 @@ public void addExternalClaim(ExternalClaim externalClaim, String tenantDomain) t
             throw new ClaimMetadataClientException(ERROR_CODE_INVALID_EXTERNAL_CLAIM_DIALECT);
         }
 
+        String claimURIRegex = IdentityClaimManagementServiceDataHolder.getInstance()
```

Comment:
let's do a null check for `IdentityClaimManagementServiceDataHolder.getInstance().getClaimConfig()`